framework/CHANGELOG.md

```diff
@@ -9,8 +9,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
+- Usage fee query for the dex adapter
```

Comment:
I think we should separate the module changelog from the framework changelog because we can version them differently going forward.